src/systems/physics/Physics.cc

```diff
@@ -1563,6 +1563,111 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
         return true;
       });
 
+  // Update link angular velocity
+  _ecm.Each<components::Link, components::AngularVelocityCmd>(
```

Comment:
Should we clear these commands at the end of `UpdateSim` like we do for `JointVelocityCmd`, so the command is not continuously applied every iteration?

Reply:
yeah good point, cleared cmds. b9e6779

Reply:
added test in b9e6779